### ISSUE TITLE
Hangs at installing packages when Avast or AVG are installed

### ISSUE THREAD

**Issue by pauby** [chocolatey/choco#3049]
### Checklist

- [X] I have verified this is the correct repository for opening this issue.
- [X] I have verified no other issues exist related to my problem.
- [X] I have verified this is not an issue for a specific package.
- [X] I have verified this issue is not security related.

### What You Are Seeing?

When the latest (as of 26 Feb 2023) version of Avast or AVG is installed, installing / upgrading packages with Chocolatey CLI hangs at `<PACKAGENAME> package files install completed. Performing other installation steps.`.

AVG installed:

![image](https://user-images.githubusercontent.com/12760779/221432900-826f64f1-9dcd-4394-bdec-aab27315af34.png)

Avast installed:

![image](https://user-images.githubusercontent.com/12760779/221433039-9eed00a1-2f0c-46d3-b734-478b0f21a2a0.png)

Note that in the above screenshots that the `chocolatey-compatibility.extension` and the `chocolatey-core.extension` do not execute any PowerShell scripts, but they have been installed. This would indicate the issue is when starting the PowerShell Host to run the Firefox package PowerShell scripts.

Neither AVG nor Avast alert the user or add to the logs that any detection / malicious behaviour is detected. This therefore looks like a bug.

![image](https://user-images.githubusercontent.com/12760779/221433178-4267c118-9e7a-4d2b-9391-fe9977c990bf.png)

![image](https://user-images.githubusercontent.com/12760779/221435831-dfbdf71c-5e4f-4d14-9fb4-10350a48bf5b.png)

This issue is being created to keep track of this issue and to highlight it to users.

### What is Expected?

Chocolatey CLI can install and upgrade packages as normal.

### How Did You Get This To Happen?

I used this [Vagrant box](https://app.vagrantup.com/StefanScherer/boxes/windows_10/versions/2021.12.09) that hasn't had any updates applied since 9 December 2021. I also used the same Vagrant box and patched it to 17 February 2023 with the same results.

1. I disabled Windows Updates
2. Installed `avastfreeantivirus` package using Chocolatey CLI.
3. Rebooted.
4. Tried to install Firefox with Chocolatey CLI using `choco install firefox` which just sat there showing the same error that we've seen. 

I also reproduced this with the package `avgantivirusfree` package (in step 2) with the same results.

### System Details

* Operating System: 10.0.19043.0
* Windows PowerShell version: 5.1.19041.1320
* Chocolatey CLI Version: 1.3.0 and 2.0.0-alpha-20230221
* Chocolatey Licensed Extension version: N/A
* Chocolatey License type: Free
* Terminal/Emulator: PowerShell Console.

### Installed Packages

```bash
Chocolatey CLI 1.3.0 only.
```

### Output Log

```bash
https://gist.github.com/pauby/421bd454f17d02e520553a5052c6da89
```

### Additional Context

We have [confirmation that the following products are affected](https://discord.com/channels/778552361454141460/1075762919704371221):

* AVG Free (version 23.1.3269 (build 23.1.7883.774)
* Avast Free (version 23.1.6049 (build 23.1.7883.774)
* Avast Pro (this was mentioned as having an issue, but I can no longer find the confirmation).

[Avast acquired AVG in 2016](https://press.avast.com/avast-announces-agreement-to-acquire-avg-for-13b) and, from a conversation on Discord, I understand they are moving to use the same underlying technology (which would make sense for them as a company and the build number on the products above lends some credence to that). This would explain why both of them have this issue. [Norton merged with Avast in 2021](https://blog.avast.com/avast-nortonlifelock-merge), but we have had no reports of any Norton product being affected.

The following issues are related:

* #3034 
* #3042 
* #3040 

Other places where issues have been reported:

* [Discord - Chocolatey CLI freezing](https://discord.com/channels/778552361454141460/1075762919704371221)
* [Reddit - I can't install packages with chocolatey](https://www.reddit.com/r/chocolatey/comments/119f83v/i_cant_install_packages_with_chocolatey/)
* [Reddit - Install getting hung up](https://www.reddit.com/r/chocolatey/comments/114b1mf/install_getting_hung_up/)

Issue has been raised over on the [Avast forums](https://forum.avast.com/index.php?topic=322819.0).

**Comment by TimKras** [comment]
A workaround is to add an axclusion in Avast:
![image](https://user-images.githubusercontent.com/5257192/222124627-7cebe58f-0238-4bfd-bfc6-1499b2f4cb13.png)

**Comment by PHR16384** [comment]
Seems like the issue is fixed?  Just tried `cup chocolatey` once more today, and it finally worked (and automatically fixed its sort-of-broken installation too).  I didn't need to uninstall Avast, or add an exception, or do any other hax or workarounds on my part as far as I'm aware.

Guessing that Avast fixed it themselves in a virus-definitions update sometime this week?

**Comment by pauby** [comment]
Thanks for confirming its working again. 

Following the same steps above (except I didn't disable Windows Update this time) I was able to confirm, on the _same_ Windows 10 VM I used, that it's now working under Avast and AVG:

1. Installed `avastfreeantivirus` / `avgantivirusfree` packages using Chocolatey CLI.
2. Rebooted.
3. Tried to install Firefox with Chocolatey CLI using `choco install firefox` and it installed.

The versions have been updated.

Avast:

![image](https://user-images.githubusercontent.com/12760779/222980621-f71ebf51-d161-4e65-a96d-db55826013b7.png)

AVG:

![image](https://user-images.githubusercontent.com/12760779/222980878-16b92384-2002-4a8d-946a-18536cc8d71a.png)

Disappointingly, Avast have not responded to the[ forum post](https://forum.avast.com/index.php?topic=322819.0) I created.

As we both confirm it's fixed, I'll go ahead and close this now and remove it as a pinned issue.
